### PR TITLE
Release  v1.3.0

### PR DESCRIPTION
**Pull request by kbasv**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

Release 1.3.0

**What testing is done?**

### CODE REVIEW

File: charts/aws-efs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
-version: 2.0.0
-appVersion: 1.2.1
+version: 2.1.0
```
Comment (wongma7) [inline charts/aws-efs-csi-driver/Chart.yaml:3]: we have helm changelog as well now, release process will need a slight update. https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/charts/aws-efs-csi-driver/CHANGELOG.md

File: docs/README.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ The following sections are Kubernetes specific. If you are a Kubernetes user, us
 | AWS EFS CSI Driver \ Kubernetes Version| maturity | v1.11 | v1.12 | v1.13 | v1.14 | v1.15 | v1.16 | v1.17+ |
 |----------------------------------------|----------|-------|-------|-------|-------|-------|-------|-------|
 | master branch                          | GA       | no    | no    | no    | no    | no    | no    | yes   |
+| v1.3.x                                 | GA       | no    | no    | no    | no    | no    | no    | yes   |
```
Comment (wongma7) [inline docs/README.md:58]: also update this line down below


  

  kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.2"

to

kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.3"

File: Makefile [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
 PKG=github.com/kubernetes-sigs/aws-efs-csi-driver
 IMAGE?=amazon/aws-efs-csi-driver
-VERSION=v1.2.1-dirty
+VERSION=v1.3.0-dirty
```
Comment (wongma7) [inline Makefile:18]: we can solve it later but seems weird how we always append dirty? i understand 'dirty' to mean it's a build from a git repo that has uncommitted changes

Comment (kbasv) [reply Makefile:18]: Should we get rid of `dirty`?

File: docs/README.md (hunk 2 above)
Comment (kbasv) [reply docs/README.md:58]: Done

File: Makefile (hunk 3 above)
Comment (wongma7) [reply Makefile:18]: mmmmmmm I guess that's a quick fix. doesn't seem valuable.

**Pull request merged**